Jenkinsfile

```diff
@@ -79,6 +70,14 @@ elifePipeline {
             }
         }
 
+        stage 'Browser Tests', {
+            actions['test:browser'] = {
+                withCommitStatus({
+                    sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=test PGDATABASE=test_browser docker-compose -f docker-compose.yml -f docker-compose.ci.yml run -p 10081:10081 --rm --name elife-xpub_app_test_browser app bash -c 'socat -d tcp-listen:10081,reuseaddr,fork tcp:localhost:10080 & npm run test:browser -- --screenshots /tmp/screenshots --screenshots-on-fails'"
+                }, 'test:browser', commit)
```

Comment:
you also need the setup around it, since this command assumes a database started and ready:
```
            try {
                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
                // do things
            } finally {
                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
            }
```

Reply:
also, the `archiveArtifacts` in `finally` is missing, which publishes the screenshots on the Jenkins UI

Reply:
Ah we removed that ... @damnhipster says they don't work anyway :)